internal/pkg/model/manifest.go

```diff
@@ -292,3 +294,11 @@ func (c *ConfigManifest) AddRelation(relation Relation) {
 func (r *ConfigRowManifest) AddRelation(relation Relation) {
 	r.Relations.Add(relation)
 }
+
+func (r *ConfigManifest) GetMetadata() *orderedmap.OrderedMap {
+	return r.Metadata
+}
```

Comment:
Myslim, ze tato metoda je zbytocna. Metadata su public,
Skor by som navrhoval metodu `MetadataMap map[string]string`, aby si nemusel mat `for` v `MapAfterLocalLoad`.